pyproject.toml

```diff
@@ -55,6 +55,7 @@ extend-select = [
   "N",           # pep8-naming
   "ARG",         # flake8-unused-arguments
   "C4",          # flake8-comprehensions
+  "E5",          # pycodestyle
```

Comment:
And drop this one.

Reply:
Done.

I'd prefer a limit (same as for code), but oh well.